src/pages/VideoCourse.vue

```diff
@@ -1,102 +1,75 @@
 <template>
-  <div class="host">
-    <b-container>
-      <b-breadcrumb :items="items" />
-      <b-row>
-        <b-col
-          md="6"
-          sm="12"
-        />
-        <b-col
-          md="6"
-          sm="12"
-        >
-          <Checkbox
-            v-if="codeEditorURL"
-            id="isautoSync"
-            label="Auto sync code with video"
-            :is-checked="isAutoSync"
-            :on-click="toggleIsAutoSync"
-          />
-        </b-col>
-      </b-row>
-      <b-row>
-        <b-col
-          md="6"
-          sm="12"
-        >
-          <youtube
-            ref="youtube"
-            :video-id="topic.videoUrl"
-            width="100%"
-            height="400"
-            :player-vars="playerVars"
-          />
-          <!-- <button @click="getCurrentTime">Get Current Time</button> -->
-        </b-col>
-        <b-col
-          md="6"
-          sm="12"
-        >
-          <CodeEditor
-            v-if="codeEditorURL"
-            :url="codeEditorURL"
-          />
-          <span v-else>No code available for this topic</span>
-        </b-col>
-      </b-row>
-      <b-row>
-        <b-col md="6">
-          <button
-            v-if="previousLink"
-            @click="onPrevious()"
-          >
-            Previous
-          </button>
-        </b-col>
-        <b-col md="6">
-          <button
-            v-if="nextLink"
-            style="float: right"
-            @click="onNext()"
-          >
-            Next
-          </button>
-        </b-col>
-      </b-row>
-      <b-row
-        v-if="!hideComments"
-        class="comment-section"
-      >
-        <b-col md="1" />
-        <b-col md="10">
-          <h1>Questions and Answers</h1>
-          <add-comment
-            ref="addcomment"
-            :comment-id="commentId"
-            :on-save="saveComment"
-            :on-cancel="cancelComment"
-            :show-rating="showRating"
-            :parent-id="topic._id"
-            class="mt-1"
-          />
-          <Comment
-            v-for="(comment, index) in comments"
-            :key="comment._id"
-            :index="index"
-            :comment-id="comment._id"
-            :comment="comment"
-            :show-rating="showRating"
-            :allow-reply="allowReply"
-            :on-delete="deleteComment"
-            :on-edit="editComment"
-            :on-save="saveComment"
-          />
-        </b-col>
-        <b-col md="2" />
-      </b-row>
-    </b-container>
-  </div>
+	<div class="host">
```

Comment:
can you please pull latest and check that there are no formatting related changes? ideally it should show only the changes done by you